spec/features/admin_csv_download_spec.rb

```diff
@@ -0,0 +1,43 @@
+require "rails_helper"
+
+RSpec.feature "Download CSV of claims" do
+  context "User is logged in" do
+    before do
+      visit admin_path
+      click_on "Sign in"
+    end
+
+    context "Completed claims exist" do
+      let!(:complete_claims) { create_list(:tslr_claim, 5, :eligible_and_submittable, submitted_at: DateTime.now) }
+      let!(:incomplete_claims) { create_list(:tslr_claim, 2, :eligible_and_submittable) }
+
+      context "User downloads CSV" do
+        before do
+          click_on "Download claims"
+        end
+
+        let(:csv) { CSV.parse(body) }
+
+        scenario "User gets a CSV" do
+          expect(page.response_headers["Content-Type"]).to eq("text/csv")
+        end
+
+        scenario "CSV has the correct number of rows" do
+          expect(csv.count).to eq(complete_claims.count + 1)
+        end
+      end
+    end
+  end
+
+  context "User is not logged in" do
+    context "User tries to download claims" do
```

Comment:
here too: we can avoid the nested contexts and have a single scenario: "cannot download claims CSV".